modules/CPDoS.py

```diff
@@ -26,50 +28,58 @@ def crawl_files(url, s, req_main, domain, custom_header, authent):
                 if len(url.split("/")) > 4:
                     url = f"{'/'.join(url.split('/')[:3])}/"
                 uri = f"{url}{fu[0]}"
-                if uri.startswith('https://'):
+                if uri.startswith("https://"):
                     uri = f"https://{uri[8:].replace('//', '/')}"
-                elif uri.startswith('http://'):
+                elif uri.startswith("http://"):
                     uri = f"https://{uri[7:].replace('//', '/')}"
 
-                #print(uri)
+                # print(uri)
                 run_cpdos_modules(uri, s, req_main, domain, custom_header, authent)
-    except:
-        #traceback.print_exc()
-        pass
-
+    except Exception as e:
+        logger.exception(e)
 
 
 def run_cpdos_modules(url, s, req_main, domain, custom_header, authent):
-    uri = "{}?CPDoS={}".format(url, random.randint(1, 100), random.randint(1, 100))
-    headers = {'User-agent': 'Mozilla/5.0 (Windows NT 6.3; WOW64; Trident/7.0; LCJB; rv:11.0) like Gecko'}
+    uri = f"{url}?CPDoS={random.randint(1, 100)}"
+    headers = {
+        "User-agent": "Mozilla/5.0 (Windows NT 6.3; WOW64; Trident/7.0; LCJB; rv:11.0) like Gecko"
+    }
     try:
-        req_main = s.get(url, headers=headers, verify=False, allow_redirects=False, timeout=15, auth=authent)
-        #print(req_main.content)
+        req_main = s.get(
+            url,
+            headers=headers,
+            verify=False,
+            allow_redirects=False,
+            timeout=15,
+            auth=authent,
+        )
+        logger.debug(req_main.content)
```

Comment:
Make the debug statements explicit via the logger use. These won't be seen by the users based on the configuration.